app/models/spree/payment_decorator.rb

```diff
@@ -73,6 +75,15 @@ def refund!(refund_amount=nil)
       end
     end
 
+    # Import from future Spree
```

Comment:
please, specify from which version so we know when upgrading.

Reply:
Resolved